Associated Template Variable - Add docs (#9702)
DataDog/documentation: content/en/dashboards/template_variables.md

{{< img src="dashboards/template_variables/selecting_template_variables.png" alt="Selecting template variables"  style="width:75%;">}}

> #### Associated Template Variables

Reviewer: ```suggestion
#### Associated template variables
```

> When selecting a template variable value, the **Associated Values** and **Other Values** sections are displayed. Associated values are calculated by taking into  the other template variable values selected on the page, and seamlessly identifies the related values without any configuration. 

Reviewer: @MirandaKapin Hi, it still looks like there's a word missing in the second sentence? I can't guess at what it is from the context given. Should it be "consideration"?

> When selecting a template variable value, you will see two sections for **Associated Values** and **Other Values**. Associated Values are calculated by taking into  the other template variable values selected on the page, and seamlessly identifies the related values without any configuration. 

Reviewer: ```suggestion
When selecting a template variable value, the **Associated Values** and **Other Values** sections are displayed. Associated values are calculated by taking into  the other template variable values selected on the page, and seamlessly identifies the related values without any configuration. 
```

@MirandaKapin is there a word missing in the second sentence? Also, what does "other values" refer to? seems like only associated values are being explained here.
